susemanager/src/mgr_bootstrap_data.py

```diff
@@ -1106,6 +1106,10 @@
         'PDID' : [2001], 'PKGLIST' : PKGLIST15_SALT + PKGLIST15_X86_ARM,
         'DEST' : '/srv/www/htdocs/pub/repositories/opensuse/15/2/bootstrap/'
     },
+    'openSUSE-Leap-15.2-x86_64-uyuni' : {
+        'BASECHANNEL' : 'opensuse_leap15_2-x86_64', 'PKGLIST' : PKGLIST15_TRAD + PKGLIST15_SALT + PKGLIST15SP0SP1_SALT + PKGLIST15_X86_ARM,
```

Comment:
we use the same base channel? I didn't check explicit with spacewalk-common-channels.ini